packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +195,16 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.actions], the intended use for the return value of this method.
   List<Widget> buildActions(BuildContext context);
 
+  /// Widget to display across the bottom of the [AppBar].
+  ///
+  /// Returns null if no widget is set.
```

Comment:
Returns null by default, i.e. a bottom widget is not included.

Reply:
Updated!